File: frontend/src/app/utils/auth/dtos/ChangedPasswordDto.ts
export class ChangedPasswordDto {
    private oldPassword: string;
    private newPassword: string;
    private newPasswordRetyped: string;

    constructor(oldPassword: string, newPassword: string, newPasswordRetyped: string) {
        this.oldPassword = oldPassword;
        this.newPassword = newPassword;
        this.newPasswordRetyped = newPasswordRetyped;
    }

}

File: pki-app/src/main/java/bsep/pkiapp/dto/ChangedPasswordDto.java
package bsep.pkiapp.dto;

public class ChangedPasswordDto {

    private String newPassword;

    private String newPasswordRetyped;

    private String oldPassword;

    public ChangedPasswordDto() {
    }

    public String getNewPassword() {
        return newPassword;
    }

    public void setNewPassword(String newPassword) {
        this.newPassword = newPassword;
    }

    public String getNewPasswordRetyped() {
        return newPasswordRetyped;
    }

    public void setNewPasswordRetyped(String newPasswordRetyped) {
        this.newPasswordRetyped = newPasswordRetyped;
    }

    public String getOldPassword() { return oldPassword; }

    public void setOldPassword(String oldPassword) { this.oldPassword = oldPassword; }
}
